README.md

```diff
@@ -15,7 +15,7 @@ Make sure you got [HPNL](https://github.com/Intel-bigdata/HPNL) installed.
 
 ```shell
 git clone https://github.com/Intel-bigdata/Spark-PMoF.git
-cd Spark-PMoF; mvn package
+cd Spark-PMoF; mvn package -DskipTests
```

Comment:
It is recommended to modify, we generally do not install Pmem in the compile environment, we need to skip the test.

Reply:
I think you want to say "If the pmem hardware is NOT ready",